### ISSUE TITLE
Typo in usage.c

### ISSUE THREAD

**Issue by jeffmcjunkin** [hashcat/hashcat#495]
Option --bitmap-max shows "--bitmap-min=24" in example:

```
$ hashcat --version
v3.10-109-gac77ee5

$ hashcat --help | grep bitmap
  --bitmap-min  | Num  | Sets minimum bits allowed for bitmaps to X  | --bitmap-min=24
  --bitmap-max  | Num  | Sets maximum bits allowed for bitmaps to X  | --bitmap-min=24
```

**Comment by jeffmcjunkin** [comment]
Fixed in #496

**Comment by jsteube** [comment]
Merged
